sefaria/views.py

```diff
@@ -401,13 +401,13 @@ def title_regex_api(request, titles, json_response=True):
         return jsonResponse({"error": "Unsupported HTTP method."}) if json_response else {"error": "Unsupported HTTP method."}
 
 
-def bundle_many_texts(refs, useTextFamily=False, as_sized_string=False, min_char=None, max_char=None, translation_language_preference=None, english_version=None, hebrew_version=None):
+def bundle_many_texts(refs, useTextFamily=0, as_sized_string=False, min_char=None, max_char=None, translation_language_preference=None, english_version=None, hebrew_version=None):
```

Comment:
I think we need to find another solution because it's not conventional to have a python function take 1 and 0 in the place of a boolean. This is especially true when there's another parameter (`as_sized_string`) that is a boolean.

Reply:
Thanks @Noah - 

I just pushed a revision, where `None` is still the default value, but inside the `if` it still checks if `useTextFamily == 1` as opposed to before where it just checked if it was not `None` (thus breaking the expected boolean functionality from the API, passing 0/1 as a boolean). 

What do you think?